test/extended/testdata/bindata.go

```diff
@@ -19587,7 +19599,8 @@ var _examplesQuickstartsDancerMysqlPersistentJson = []byte(`{
   },
   "message": "The following service(s) have been created in your project: ${NAME}, ${DATABASE_SERVICE_NAME}.\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/openshift/dancer-ex/blob/master/README.md.",
   "labels": {
-    "template": "dancer-mysql-persistent"
+    "template": "dancer-mysql-persistent",
+    "app": "dancer-mysql-persistent"
```

Comment:
The `app` label should be parameterized so that when templates are used in conjunction to form a single application we can make the `app` labels match.